packages/react-core/src/components/InlineEdit/InlineEditToggle.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/InlineEdit/inline-edit';
+import { css } from '@patternfly/react-styles';
+import { Button } from '../Button';
+import PencilAltIcon from '@patternfly/react-icons/dist/esm/icons/pencil-alt-icon';
+
+export interface InlineEditToggleProps {
+  /** Action to perform on the edit icon click. */
+  onToggle?: (event: React.MouseEvent) => void;
+}
+
+export const InlineEditToggle: React.FunctionComponent<InlineEditToggleProps> = ({
+  onToggle
+}: InlineEditToggleProps) => (
+  <div className={css(styles.inlineEditAction, styles.modifiers.enableEditable)}>
+    <Button variant="plain" aria-label="Edit" onClick={onToggle}>
+      <PencilAltIcon />
```

Comment:
@lboehling similar question as the save/cancel button icons: do we want to support custom icon here? And do we also want to allow either using text instead ("Edit") or using both text and icon as seen in the [full page edit design guidelines](https://www.patternfly.org/components/inline-edit/design-guidelines#full-page-edit)?